python313Packages.atopile: 0.11.2 -> 0.12.4

Sigmanificient opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

Sigmanificient committed:
python313Packages.atopile: 0.11.2 -> 0.11.6

LunNova commented:
Can't build/test this due to failing deps:

pygls: `E   AttributeError: module 'lsprotocol.types' has no attribute 'TextDocumentContentChangeEvent_Type1'`

Sigmanificient commented:
Oh no, the `lsprotocols` bumps also broke atopile via `pygls` :disappointed:

GaetanLepage committed:
python3Packages.questionary: skip failing test

GaetanLepage committed:
python3Packages.atopile-easyeda2kicad: 0.9.6 -> 0.9.7

Diff: https://github.com/atopile/easyeda2kicad.py/compare/v0.9.6...v0.9.7

Changelog: https://github.com/atopile/easyeda2kicad.py/releases/tag/v0.9.7

GaetanLepage committed:
python3Packages.atopile: 0.11.6 -> 0.12.4

Diff: https://github.com/atopile/atopile/compare/v0.11.6...v0.12.4

Changelog: https://github.com/atopile/atopile/releases/tag/v0.12.4

GaetanLepage commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 435668`
Commit: `69696725ab64c96f52f9517172971edb63eaa4ca`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 6 packages marked as broken and skipped:</summary>
  <ul>
    <li>maubot</li>
    <li>maubot.dist</li>
    <li>python312Packages.maubot</li>
    <li>python312Packages.maubot.dist</li>
    <li>python313Packages.maubot</li>
    <li>python313Packages.maubot.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 6 packages failed to build:</summary>
  <ul>
    <li>commitizen</li>
    <li>commitizen.dist</li>
    <li>python312Packages.qiling</li>
    <li>python312Packages.qiling.dist</li>
    <li>python313Packages.qiling</li>
    <li>python313Packages.qiling.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 54 packages built:</summary>
  <ul>
    <li>atopile (python313Packages.atopile)</li>
    <li>atopile.dist (python313Packages.atopile.dist)</li>
    <li>audible-cli</li>
    <li>audible-cli.dist</li>
    <li>beatprints</li>
    <li>beatprints.dist</li>
    <li>bump-my-version (python313Packages.bump-my-version)</li>
    <li>bump-my-version.dist (python313Packages.bump-my-version.dist)</li>
    <li>copier (python313Packages.copier)</li>
    <li>copier.dist (python313Packages.copier.dist)</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>myjwt (python313Packages.myjwt)</li>
    <li>myjwt.dist (python313Packages.myjwt.dist)</li>
    <li>phase-cli</li>
    <li>phase-cli.dist</li>
    <li>python312Packages.atopile-easyeda2kicad</li>
    <li>python312Packages.atopile-easyeda2kicad.dist</li>
    <li>python312Packages.bentoml</li>
    <li>python312Packages.bentoml.dist</li>
    <li>python312Packages.bump-my-version</li>
    <li>python312Packages.bump-my-version.dist</li>
    <li>python312Packages.copier</li>
    <li>python312Packages.copier-template-tester</li>
    <li>python312Packages.copier-template-tester.dist</li>
    <li>python312Packages.copier.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.myjwt</li>
    <li>python312Packages.myjwt.dist</li>
    <li>python312Packages.mypy-boto3-builder</li>
    <li>python312Packages.mypy-boto3-builder.dist</li>
    <li>python312Packages.questionary</li>
    <li>python312Packages.questionary.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python313Packages.atopile-easyeda2kicad</li>
    <li>python313Packages.atopile-easyeda2kicad.dist</li>
    <li>python313Packages.bentoml</li>
    <li>python313Packages.bentoml.dist</li>
    <li>python313Packages.copier-template-tester</li>
    <li>python313Packages.copier-template-tester.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.mypy-boto3-builder</li>
    <li>python313Packages.mypy-boto3-builder.dist</li>
    <li>python313Packages.questionary</li>
    <li>python313Packages.questionary.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>tockloader</li>
    <li>tockloader.dist</li>
    <li>trak</li>
    <li>trak.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 6 packages marked as broken and skipped:</summary>
  <ul>
    <li>maubot</li>
    <li>maubot.dist</li>
    <li>python312Packages.maubot</li>
    <li>python312Packages.maubot.dist</li>
    <li>python313Packages.maubot</li>
    <li>python313Packages.maubot.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>commitizen</li>
    <li>commitizen.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.qiling</li>
    <li>python312Packages.qiling.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python313Packages.qiling</li>
    <li>python313Packages.qiling.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 50 packages built:</summary>
  <ul>
    <li>atopile (python313Packages.atopile)</li>
    <li>atopile.dist (python313Packages.atopile.dist)</li>
    <li>audible-cli</li>
    <li>audible-cli.dist</li>
    <li>beatprints</li>
    <li>beatprints.dist</li>
    <li>bump-my-version (python313Packages.bump-my-version)</li>
    <li>bump-my-version.dist (python313Packages.bump-my-version.dist)</li>
    <li>copier (python313Packages.copier)</li>
    <li>copier.dist (python313Packages.copier.dist)</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>myjwt (python313Packages.myjwt)</li>
    <li>myjwt.dist (python313Packages.myjwt.dist)</li>
    <li>phase-cli</li>
    <li>phase-cli.dist</li>
    <li>python312Packages.atopile-easyeda2kicad</li>
    <li>python312Packages.atopile-easyeda2kicad.dist</li>
    <li>python312Packages.bentoml</li>
    <li>python312Packages.bentoml.dist</li>
    <li>python312Packages.bump-my-version</li>
    <li>python312Packages.bump-my-version.dist</li>
    <li>python312Packages.copier</li>
    <li>python312Packages.copier-template-tester</li>
    <li>python312Packages.copier-template-tester.dist</li>
    <li>python312Packages.copier.dist</li>
    <li>python312Packages.myjwt</li>
    <li>python312Packages.myjwt.dist</li>
    <li>python312Packages.mypy-boto3-builder</li>
    <li>python312Packages.mypy-boto3-builder.dist</li>
    <li>python312Packages.questionary</li>
    <li>python312Packages.questionary.dist</li>
    <li>python313Packages.atopile-easyeda2kicad</li>
    <li>python313Packages.atopile-easyeda2kicad.dist</li>
    <li>python313Packages.bentoml</li>
    <li>python313Packages.bentoml.dist</li>
    <li>python313Packages.copier-template-tester</li>
    <li>python313Packages.copier-template-tester.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.mypy-boto3-builder</li>
    <li>python313Packages.mypy-boto3-builder.dist</li>
    <li>python313Packages.questionary</li>
    <li>python313Packages.questionary.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>tockloader</li>
    <li>tockloader.dist</li>
    <li>trak</li>
    <li>trak.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>commitizen</summary>
<pre>no configure script, doing nothing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built commitizen-4.9.1-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/commitizen-4.9.1...OK
Repacking wheel as ./commitizen-4.9.1-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;pythonRuntimeDepsCheckHook&quot; }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for commitizen-4.9.1-py3-none-any.whl
  - prompt-toolkit!=3.0.52 not satisfied by version 3.0.52</pre>
</details>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>commitizen</summary>
<pre>Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built commitizen-4.9.1-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/commitizen-4.9.1...OK
Repacking wheel as ./commitizen-4.9.1-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for commitizen-4.9.1-py3-none-any.whl
  - prompt-toolkit!=3.0.52 not satisfied by version 3.0.52</pre>
</details>
</details>

GaetanLepage commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 435668`
Commit: `69696725ab64c96f52f9517172971edb63eaa4ca`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 6 packages marked as broken and skipped:</summary>
  <ul>
    <li>maubot</li>
    <li>maubot.dist</li>
    <li>python312Packages.maubot</li>
    <li>python312Packages.maubot.dist</li>
    <li>python313Packages.maubot</li>
    <li>python313Packages.maubot.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 6 packages failed to build:</summary>
  <ul>
    <li>commitizen</li>
    <li>commitizen.dist</li>
    <li>python312Packages.qiling</li>
    <li>python312Packages.qiling.dist</li>
    <li>python313Packages.qiling</li>
    <li>python313Packages.qiling.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 54 packages built:</summary>
  <ul>
    <li>atopile (python313Packages.atopile)</li>
    <li>atopile.dist (python313Packages.atopile.dist)</li>
    <li>audible-cli</li>
    <li>audible-cli.dist</li>
    <li>beatprints</li>
    <li>beatprints.dist</li>
    <li>bump-my-version (python313Packages.bump-my-version)</li>
    <li>bump-my-version.dist (python313Packages.bump-my-version.dist)</li>
    <li>copier (python313Packages.copier)</li>
    <li>copier.dist (python313Packages.copier.dist)</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>myjwt (python313Packages.myjwt)</li>
    <li>myjwt.dist (python313Packages.myjwt.dist)</li>
    <li>phase-cli</li>
    <li>phase-cli.dist</li>
    <li>python312Packages.atopile-easyeda2kicad</li>
    <li>python312Packages.atopile-easyeda2kicad.dist</li>
    <li>python312Packages.bentoml</li>
    <li>python312Packages.bentoml.dist</li>
    <li>python312Packages.bump-my-version</li>
    <li>python312Packages.bump-my-version.dist</li>
    <li>python312Packages.copier</li>
    <li>python312Packages.copier-template-tester</li>
    <li>python312Packages.copier-template-tester.dist</li>
    <li>python312Packages.copier.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.myjwt</li>
    <li>python312Packages.myjwt.dist</li>
    <li>python312Packages.mypy-boto3-builder</li>
    <li>python312Packages.mypy-boto3-builder.dist</li>
    <li>python312Packages.questionary</li>
    <li>python312Packages.questionary.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python313Packages.atopile-easyeda2kicad</li>
    <li>python313Packages.atopile-easyeda2kicad.dist</li>
    <li>python313Packages.bentoml</li>
    <li>python313Packages.bentoml.dist</li>
    <li>python313Packages.copier-template-tester</li>
    <li>python313Packages.copier-template-tester.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.mypy-boto3-builder</li>
    <li>python313Packages.mypy-boto3-builder.dist</li>
    <li>python313Packages.questionary</li>
    <li>python313Packages.questionary.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>tockloader</li>
    <li>tockloader.dist</li>
    <li>trak</li>
    <li>trak.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 6 packages marked as broken and skipped:</summary>
  <ul>
    <li>maubot</li>
    <li>maubot.dist</li>
    <li>python312Packages.maubot</li>
    <li>python312Packages.maubot.dist</li>
    <li>python313Packages.maubot</li>
    <li>python313Packages.maubot.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>commitizen</li>
    <li>commitizen.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.qiling</li>
    <li>python312Packages.qiling.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python313Packages.qiling</li>
    <li>python313Packages.qiling.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 50 packages built:</summary>
  <ul>
    <li>atopile (python313Packages.atopile)</li>
    <li>atopile.dist (python313Packages.atopile.dist)</li>
    <li>audible-cli</li>
    <li>audible-cli.dist</li>
    <li>beatprints</li>
    <li>beatprints.dist</li>
    <li>bump-my-version (python313Packages.bump-my-version)</li>
    <li>bump-my-version.dist (python313Packages.bump-my-version.dist)</li>
    <li>copier (python313Packages.copier)</li>
    <li>copier.dist (python313Packages.copier.dist)</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>myjwt (python313Packages.myjwt)</li>
    <li>myjwt.dist (python313Packages.myjwt.dist)</li>
    <li>phase-cli</li>
    <li>phase-cli.dist</li>
    <li>python312Packages.atopile-easyeda2kicad</li>
    <li>python312Packages.atopile-easyeda2kicad.dist</li>
    <li>python312Packages.bentoml</li>
    <li>python312Packages.bentoml.dist</li>
    <li>python312Packages.bump-my-version</li>
    <li>python312Packages.bump-my-version.dist</li>
    <li>python312Packages.copier</li>
    <li>python312Packages.copier-template-tester</li>
    <li>python312Packages.copier-template-tester.dist</li>
    <li>python312Packages.copier.dist</li>
    <li>python312Packages.myjwt</li>
    <li>python312Packages.myjwt.dist</li>
    <li>python312Packages.mypy-boto3-builder</li>
    <li>python312Packages.mypy-boto3-builder.dist</li>
    <li>python312Packages.questionary</li>
    <li>python312Packages.questionary.dist</li>
    <li>python313Packages.atopile-easyeda2kicad</li>
    <li>python313Packages.atopile-easyeda2kicad.dist</li>
    <li>python313Packages.bentoml</li>
    <li>python313Packages.bentoml.dist</li>
    <li>python313Packages.copier-template-tester</li>
    <li>python313Packages.copier-template-tester.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.mypy-boto3-builder</li>
    <li>python313Packages.mypy-boto3-builder.dist</li>
    <li>python313Packages.questionary</li>
    <li>python313Packages.questionary.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>tockloader</li>
    <li>tockloader.dist</li>
    <li>trak</li>
    <li>trak.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>commitizen</summary>
<pre>no configure script, doing nothing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built commitizen-4.9.1-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/commitizen-4.9.1...OK
Repacking wheel as ./commitizen-4.9.1-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;pythonRuntimeDepsCheckHook&quot; }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for commitizen-4.9.1-py3-none-any.whl
  - prompt-toolkit!=3.0.52 not satisfied by version 3.0.52</pre>
</details>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>commitizen</summary>
<pre>Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built commitizen-4.9.1-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/commitizen-4.9.1...OK
Repacking wheel as ./commitizen-4.9.1-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for commitizen-4.9.1-py3-none-any.whl
  - prompt-toolkit!=3.0.52 not satisfied by version 3.0.52</pre>
</details>
</details>
